Sources/Plasma/Apps/plClient/Mac-Cocoa/main.mm

```diff
@@ -107,6 +108,7 @@ @interface AppDelegate : NSWindowController <NSApplicationDelegate,
    @public
     plClientLoader gClient;
     dispatch_source_t _displaySource;
+    std::shared_ptr<plMacDisplayHelper> _displayHelper;
```

Comment:
Admittedly, I have an unnatural distaste for `std::shared_ptr`, but the display helper doesn't actually seem to be shared. This should probably be at most `std::unique_ptr`  or not a pointer at all, like `gClient`.